### PR TITLE
Add arbitrary content example

### PR DESCRIPTION
**Pull request by beccahughes**
***
<a href="https://pr-preview.s3.amazonaws.com/WICG/picture-in-picture/pull/135.html" title="Last updated on Apr 30, 2019, 11:50 PM UTC (3a1920b)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/WICG/picture-in-picture/135/a418f15...3a1920b.html" title="Last updated on Apr 30, 2019, 11:50 PM UTC (3a1920b)">Diff</a>

### CODE REVIEW

File: index.bs [hunk 1]
```diff
@@ -190,6 +190,32 @@ extensible.
 </script>
 ```
 
+## Display Arbitrary Content ## {#example-arbitrary-content}
+
+```html
+<div id="interactive-ui"></div>
+<div id="non-interactive-ui"></div>
```
Comment (mounirlamouri) [inline index.bs]: This ends up being rendered on one line. Should you add double spaces to have a line break?

Comment (beccahughes) [reply index.bs]: Done.

Comment (mounirlamouri) [reply index.bs]: Given that the "Preview" and "Diff" have a different rendering, I'm not sure if I was actually correct now :(

**Comment by mounirlamouri** [comment]
LGTM. Maybe it would be good to add both `<div>` into a container like:
```html
<div style="display: none;">
  <div id="interactive-ui"></div>
  <div id="non-interactive-ui"></div>
</div>
```

So it's clear they are not meant to be visible?

**Comment by beccahughes** [comment]
Thanks! I have updated this.

**Pull request merged**